Projects/Home/Sources/Viewables/CommonClaims/Collection/CommonClaimCard.swift

```diff
@@ -107,3 +108,22 @@ extension CommonClaimCard: Reusable {
         )
     }
 }
+
+extension CommonClaimCard {
+    static func journey(
+        data: GraphQL.CommonClaimsQuery.Data.CommonClaim,
+        index: TableIndex
+    ) -> some JourneyPresentation {
+        Journey(
+            CommonClaimDetail(data: data, index: index),
+            style: .detented(.medium, .large),
+            options: .defaults
+        )
+        .onAction(HomeStore.self) { action in
+            if case .openClaims = action {
+                DismissJourney()
```

Comment:
Wouldn't the correct behaviour be to present claims flow here? Doing this you depend on a state where Presentation queues the new modal and then presents it when you dismiss, making this code a bit hard to follow 🤔